osquery/tables/networking/darwin/wifi.mm

```diff
@@ -96,14 +96,14 @@ QueryData genKnownWifiNetworks(QueryContext& context) {
   std::string key;
   auto status = getKnownNetworksKey(key);
   if (!status.ok()) {
-    VLOG(1) << status.getMessage();
+    LOG(WARNING) << status.getMessage();
```

Comment:
There is no context from the call site here, I'd recommend keeping it a verbose log.